tutorials/3d/3d_rendering_limitations.rst

```diff
@@ -0,0 +1,125 @@
+
+Texture size limits
+-------------------
+
+Modern desktop/laptop GPUs can only render textures up to 16384×16384. If you
+need to display a larger texture for some reason, you'll need to split it into
+multiple smaller textures. Old GPUs like the GeForce 8/9 series are limited to
+8192×8192.
+
+Mobile GPUs are typically limited to 4096×4096 textures. Therefore, if you want
+your texture to display correctly on all platforms, you should avoid using
+textures larger than 4096×4096.
+
+2D line drawing
+---------------
+
+In OpenGL, 2D line drawing is implemented by the graphics driver. The maximum
+line thickness that can be drawn is implementation-defined. AMD and Intel
+graphics drivers only support 1 pixel-thick lines. NVIDIA graphics drivers allow
+drawing lines up to 5 pixels thick.
+
+Note that this limitation doesn't affect lines drawn by the :ref:`class_Line2D`
+node, since these are drawn using triangle strips instead of relying on OpenGL
+line drawing.
+
+Color banding
+-------------
+
+When using the GLES3 or Vulkan renderers, Godot's 3D engine renders internally
+in HDR. However, the rendering output will be tonemapped to a low dynamic range
+so it can be displayed on the screen. This can result in visible banding,
+especially when using untextured materials. This can also be seen in 2D projects
+when using smooth gradient textures.
+
+There are several ways to alleviate banding. Here are a few examples:
+
+- Bake some noise into your textures. This is mainly effective in 2D, e.g. for
+  vignetting effects.
+- Implement a debanding shader as a :ref:`screen-reading shader <doc_screen-reading_shaders>`.
+  `Godot currently doesn't provide a built-in debanding shader <https://github.com/godotengine/godot/issues/17006>`__,
+  but this may be added in a future release.
+
+.. seealso::
+
+    See `Banding in Games: A Noisy Rant <http://loopit.dk/banding_in_games.pdf>`__
+    for more details about banding and ways to combat it.
+
+Depth buffer precision
+----------------------
+
+To sort objects in 3D space, rendering engines rely on a *depth buffer*. This
+buffer has a finite precision: 24-bit on desktop platforms, sometimes 16-bit on
+mobile platforms (for performance reasons). If two different objects end up on
+the same buffer value, then Z-fighting will occur. This will materialize as
+textures flickering back and forth as the camera moves or rotates.
+
+To make the depth buffer more precise over the rendered area, you should
+*increase* the Camera node's **Near** property. However, be careful: if you set
+it too high, players will be able to see through nearby geometry.
+
+If you only need high precision when the player can see far away, you could
+change it dynamically based on the game conditions. For instance, if the player
+enters an airplane, the **Near** property can be temporarily increased to avoid
+Z-fighting in the distance. It can then be decreased once the player leaves the
+airplane.
```

Comment:
```suggestion
airplane.

Sometimes, it may also be possible to simply move the Z-fighting objects farther apart without that being visible to the player, depending on the scene.
```